UnitySDK/Assets/ML-Agents/Scripts/Agent.cs

```diff
@@ -175,15 +176,7 @@ public AgentInfo Info
 
         /// Current Agent action (message sent from Brain).
         AgentAction m_Action;
-
-        /// Represents the reward the agent accumulated during the current step.
-        /// It is reset to 0 at the beginning of every step.
-        /// Should be set to a positive value when the agent performs a "good"
-        /// action that we wish to reinforce/reward, and set to a negative value
-        /// when the agent performs a "bad" action that we wish to punish/deter.
-        /// Additionally, the magnitude of the reward should not exceed 1.0
-        float m_Reward;
-
+        
         /// Keeps track of the cumulative reward in this episode.
         float m_CumulativeReward;
```

Comment:
Should Agent.m_CumulativeReward be removed too?

Reply:
yes